tests/odm/test_fields.py

```diff
@@ -89,9 +103,31 @@ async def test_custom_filed_types():
     assert Encoder().encode(c2_fromdb) == Encoder().encode(c2)
 
 
-def test_hidden(document):
-    assert document.revision_id is None
-    assert "revision_id" not in document.dict()
+async def test_hidden(document):
+    document = await DocumentTestModel.find_one()
+
+    assert "test_list" not in document.dict()
+
+
+async def test_param_exclude(document):
```

Comment:
It is better to use pytest parametrizing here. Using it you can set input values and expected results. With it you will avoid loops and if-else stuff here.
https://docs.pytest.org/en/6.2.x/parametrize.html